app/enterprise/0.35-x/deployment/installation/ubuntu.md

```diff
@@ -0,0 +1,197 @@
+---
+title: How to Install Kong Enterprise on Ubuntu
+---
+
+## Introduction
+
+This guide walks through downloading, installing, and starting Kong Enterprise
+using Ubuntu and PostgreSQL 9.5. The configuration shown in this guide is
+intended only as an example -- you will want to modify and take additional
+measures to secure your Kong Enterprise system before moving to a production
+environment.
+
+
+## Prerequisites
+
+To complete this guide you will need:
+
+- An Ubuntu system with root equivalent access.
+- The ability to SSH to the system.
+
+
+## Step 1. Download Kong Enterprise
+
+1. Download the .deb package
+
+    Log in to [Bintray](http://bintray.com) to download the latest Kong
+    Enterprise .deb for the desired version of Ubuntu. Your **Sales** or
+    **Support** contact will email this credential to you.
+
+    Copy the file to your home directory:
+
+    ```
+    $ scp kong-enterprise-edition-0.35.xxx.xxx.deb <ubuntu user>@<serverip:~
+    ```
+
+2. Obtain your Kong Enterprise license
+
+    If you do not already have your license file, you can download it from your
+    account files in Bintray 
+    `https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`
+
+    Ensure your license file is in proper `JSON`:
+
+    ```json
+      {"license":{"signature":"91e6dd9716d12ffsn4a5ckkb16a556dbebdbc4d0a66d9b2c53f8c8d717eb93dd2bdbe2cb3ef51c20806f14345128907da35","payload":{"customer":"Kong Inc","license_creation_date":"2019-05-07","product_subscription":"Kong Enterprise Edition","admin_seats":"5","support_plan":"None","license_expiration_date":"2021-04-01","license_key":"00Q1K00000zuUAwUAM_a1V1K000005kRhuUAE"},"version":1}}
+   ```
+
+3. Securely copy the license file to the Ubuntu system
+
+    ```
+    $ scp license.json <ubuntu username>@<serverip>:~
+    ```
+
+
+## Step 2. Install Kong Enterprise
+
+1. Install Kong Enterprise
+
+    ```
+    $ sudo apt-get update
+    $ sudo apt-get install openssl libpcre3 procps perl
+    $ sudo dpkg -i kong-enterprise-edition-0.35.xxx.xxx.deb
+    ```
+  >Note: Your version may be different based on when you obtained the package
+
+2. Copy the license file to the `/etc/kong` directory
+
+    ```
+    $ sudo cp license.json /etc/kong/license.json
+    ```
+    Kong will look for a valid license in this location.
+
+
+## Step 3. Install PostgreSQL
+
+1. Install PostgreSQL
+
+    ```
+    $ sudo apt-get install postgresql postgresql-contrib
```

Comment:
pin the version here

Reply:
I've updated this to `sudo apt-get install postgresql-9.5 postgresql-contrib`